src/pages/unlock.tsx

```diff
@@ -15,6 +15,14 @@ import { useAnalytics } from '@common/hooks/analytics/use-analytics';
 import { RouteUrls } from '@routes/route-urls';
 import { useOnboardingState } from '@common/hooks/auth/use-onboarding-state';
 import { getViewMode } from '@common/utils';
+import { Caption } from '@components/typography';
+import { useWaitingMessage, WaitingMessages } from '@common/utils/use-waiting-message';
+
+const waitingMessages: WaitingMessages = {
+  '2': 'Please wait a few seconds...',
```

Comment:
```suggestion
  '2': 'Please wait a few seconds…',
```

Very minor, but this uses the actual ellipsis unicode character, rather than just three full stops